templates/jobservice/jobservice-secrets.yaml

```diff
@@ -7,5 +7,7 @@ metadata:
 type: Opaque
 data:
   JOBSERVICE_SECRET: {{ .Values.jobservice.secret | default (randAlphaNum 16) | b64enc | quote }}
+  {{- if not .Values.registry.credentials.existingSecret }}
```

Comment:
remove line 10 & line 12 as there is no `.Values.registry.credentials.existingSecret`

Reply:
"existingSecret" has been added